implementations/rust/ockam/ockam_command/src/tcp/inlet/create.rs

```diff
@@ -215,8 +214,32 @@ impl CreateCommand {
                     &cmd.to
                         .to_string()
                         .color(OckamColor::PrimaryResource.color())
-                ),
-            )
+                )
+            } else if inlet.status == ConnectionStatus::Up {
+                fmt_ok!(
+                    "TCP inlet {} on node {} is now sending traffic\n",
+                    &cmd.from
+                        .to_string()
+                        .color(OckamColor::PrimaryResource.color()),
+                    &node.node_name().color(OckamColor::PrimaryResource.color())
+                ) + &fmt_log!(
+                    "to the outlet at {}",
+                    &cmd.to
+                        .to_string()
+                        .color(OckamColor::PrimaryResource.color())
+                )
```

Comment:
I just realized from checking the screenshots that this produces a weird message due to the second line (`"to the outlet at {}",`).

Can we change it to the following?

```
fmt_ok!(
    "The inlet {} on node {} will automatically connect when the outlet at {} is available\n",
    &cmd.from
        .to_string()
        .color(OckamColor::PrimaryResource.color()),
    &node.node_name().color(OckamColor::PrimaryResource.color()),
    &cmd.to
        .to_string()
        .color(OckamColor::PrimaryResource.color())
)
```